### PR TITLE
Add additional terms for components

### PR DESCRIPTION
**Pull request by Umasankar430**

### CODE REVIEW

File: templates/data/meta-info.json [hunk 1]
```diff
@@ -156,16 +156,16 @@
...
   },
   "status.php": {
     "title": "The ARIA Status Role",
     "desc": "If part of a page is updated (e.g. because of an AJAX call), users of screen readers and other assistive tech need to be informed.  Enter the ARIA role \"status\""
   },
   "switch.php": {
-    "title": "The ARIA Switch Role",
-    "desc": "A control with no native HTML5 equivalent, switches have become popular recently in web and mobile applications, but many are inaccessible to screen reader and keyboard-only users.  Here's how to code them correctly."
+    "title": "The ARIA Switch/Toggle Role",
+    "desc": "A control with no native HTML5 equivalent, switches/Toggles have become popular recently in web and mobile applications, but many are inaccessible to screen reader and keyboard-only users.  Here's how to code them correctly."
```
Comment (alisonhall) [inline templates/data/meta-info.json:168]: We should be consistent with casing:

```suggestion
    "desc": "A control with no native HTML5 equivalent, switches/toggles have become popular recently in web and mobile applications, but many are inaccessible to screen reader and keyboard-only users.  Here's how to code them correctly."
```

File: content/body/switch.php [hunk 2]
```diff
@@ -35,8 +35,8 @@
 <p>This code is based on information from the
     <a href="https://developer.mozilla.org/en-US/">MDN</a> article on
     <a href="https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_switch_role">Using
-      the switch role</a>.  The switch reports the checked state as "on" or "off" in VoiceOver
-    and "checked" or "unchecked" in NVDA and ChromeVox.  In order to make some consistency among user agents, an <code>aria-describedby</code> on the switch can state the "on/off" state to all screen readers. This description is also given visually, to make it obvious what the state is for sighted users.  Developers could hide this text with the <a href="screen-reader-only-text.php"><code>sr-only</code> class</a>, and put "off" and "on" labels on sides of the right and left sides of the component if they wish instead. 
+      the switch/toggle role</a>.  The switch/toggle reports the checked state as "on" or "off" in VoiceOver
```
Comment (alisonhall) [inline content/body/switch.php:38]: There is no such thing as an ARIA toggle role, which is why it just said "switch" in this sentence.

```suggestion
      the switchrole</a>.  The switch/toggle reports the checked state as "on" or "off" in VoiceOver
```

File: content/body/switch.php [hunk 3]
```diff
@@ -2,13 +2,13 @@
   A switch is like a checkbox, in that is designed to be an input control that has a binary value
   (either <strong>checked</strong> and <strong>unchecked</strong> or <strong>on</strong> or <strong>off</strong>,
   depending on the screen reader).
-  Like <a href="tabs.php">tablists.php</a>, switches do not have a native HTML5 tag, so we implement custom code using
+  Like <a href="tabs.php">tablists.php</a>, switches/Toggles do not have a native HTML5 tag, so we implement custom code using
   the
-  <code>switch</code> role in JavaScript.
+  <code>switch/toggle</code> role in JavaScript.
```
Comment (alisonhall) [inline content/body/switch.php:7]: There is no such thing as an ARIA toggle role, which is why it just said "switch" in this sentence.

```suggestion
  <code>switch</code> role in JavaScript.
```

File: content/body/switch.php [hunk 4]
```diff
@@ -2,13 +2,13 @@
   A switch is like a checkbox, in that is designed to be an input control that has a binary value
   (either <strong>checked</strong> and <strong>unchecked</strong> or <strong>on</strong> or <strong>off</strong>,
   depending on the screen reader).
-  Like <a href="tabs.php">tablists.php</a>, switches do not have a native HTML5 tag, so we implement custom code using
+  Like <a href="tabs.php">tablists.php</a>, switches/Toggles do not have a native HTML5 tag, so we implement custom code using
```
Comment (alisonhall) [inline content/body/switch.php:5]: We need to be consistent with casing:

```suggestion
  Like <a href="tabs.php">tablists.php</a>, switches/toggles do not have a native HTML5 tag, so we implement custom code using
```

**Comment by alisonhall** [comment]
@Umasankar430 , I still only see page changes around the Switch/Toggle. Have you been able to look at other terms?

File: content/body/switch.php [hunk 5]
```diff
@@ -2,13 +2,13 @@
...
   the
-  <code>switch</code> role in JavaScript.
+  <code>switch/toggle</code> role in JavaScript.
 </p>
 
 <p>
-  Many developers will implement switches using the <code>&lt;input type="checkbox"&gt;</code>, since the native HTML5
+  Many developers will implement switches/toggles using the <code>&lt;input type="checkbox"&gt;</code>, since the native HTML5
```
Comment (zoltan-dulac) [inline content/body/switch.php:11]: I would like to ensure we use the word `switch` more than toggle.  Toggle should be used in the title for SEO purposes.  In the first paragraph it should be referred to once:


> A switch *(also known as a toggle)* is like a checkbox, in that is designed to be an input control that has a binary value

I would not refer to toggle elsewhere in the code.  We should be promoting the name that is used as an ARIA role wherever possible as default.

File: templates/data/meta-info.json [hunk 6]
```diff
@@ -67,7 +67,7 @@
     "isNPM": "true"
   },
   "dropdown.php": {
-    "title": "Accessible Dropdowns/Drawers/Expando",
+    "title": "Accessible Drawers",
```
Comment (zoltan-dulac) [inline templates/data/meta-info.json:70]: Please rename "Accessible Drawers (a.k.a. Accordions or Expandos)".

File: templates/data/meta-info.json [hunk 7]
```diff
@@ -67,7 +67,7 @@
     "isNPM": "true"
   },
   "dropdown.php": {
-    "title": "Accessible Dropdowns/Drawers/Expando",
+    "title": "Accessible Drawers (a.k.a. Accordions or Expandos)",
```
Comment (alisonhall) [inline templates/data/meta-info.json:70]: Where did the "Dropdown" name go?

**Review by zoltan-dulac — CHANGES_REQUESTED** [review]

File: templates/data/meta-info.json [hunk 8]
```diff
@@ -156,15 +156,15 @@
     "desc": "This page is a perfect example of why the First Rule of ARIA (i.e. don't use ARIA) is so relevant, especially for mobile screen reader users."
   },
   "spinner.php": {
-    "title": "Accessible Numeric Fields",
+    "title": "Number Input Spinner",
```
[the remark this answers is not in the document]
Comment (zoltan-dulac) [reply templates/data/meta-info.json]: I agree.  Let's change back.  Make the heading for that page consistent as well.

**Review by alisonhall — APPROVED** [review]

**Pull request merged**